### DISCUSSION TITLE
Random black screen when toggle interface

### DISCUSSION THREAD

**Discussion by Teages** [ppy/osu#17108] [Q&A] [answered]
lazer version: 2022.226.0
platform: windows 11 (22000.527)

the lazer will show a black screen when I toggle interface (start a new game play, close song list etc.) or resize window

But it still working well except showing me a black screen.

I tried to resize the window a few times and reproduced this bug. 
[network.log](https://github.com/ppy/osu/files/8190440/network.log)
[performance.log](https://github.com/ppy/osu/files/8190441/performance.log)
[updater.log](https://github.com/ppy/osu/files/8190442/updater.log)
[runtime.log](https://github.com/ppy/osu/files/8190443/runtime.log)

I tried to search issues but only found a issue about Android and it is same but marked as "missing details". 
(https://github.com/ppy/osu/issues/14498 the issues on Android)

It happens very often, every time I end the game because of this bug

**Comment by bdach** [comment]
please attach a video of how this looks. also hardware specifications of the pc may be useful.

**Reply by Teages** [reply]
Well, it is easy. resize window is not the only way to reproduced the bug. 

My pc is a laptop: Redmi G 2021, which have a i5-11260H and GTX 3050 Laptop(driver 511.23). 

https://user-images.githubusercontent.com/39217423/156881386-b0dffff4-a7c4-4f52-90b3-a6d14e9d45eb.mp4


[updater.log](https://github.com/ppy/osu/files/8190562/updater.log)
[runtime.log](https://github.com/ppy/osu/files/8190563/runtime.log)
[network.log](https://github.com/ppy/osu/files/8190564/network.log)
[performance.log](https://github.com/ppy/osu/files/8190565/performance.log)

**Reply by bdach** [reply]
looks like a possible driver error. please check if there are any driver updates for your device.

also please check whether the pc isn't experiencing low memory when this happens.

**Reply by Teages** [reply]
Yes. it is a driver error. I tried to update to the latest driver (511.79) and reboot, and it never happen again. 

And then I ran other programs to increase memory usage, still can not reproduce the problem

**Reply by Teages** [reply]
Oh, It happened again... 
I am checking the settings in home page. While I check the setting of the way to show score(In Chinese "显示分数的方式", my option is classic), it happened again.
![image](https://user-images.githubusercontent.com/39217423/157372485-8f1f12a2-2a9c-4a0c-b2d3-4185f3bf31b2.png)

It happens before 2022-03-09 04:23:45 (base on log event)
```log
2022-03-09 04:23:45 [verbose]: Game-wide working beatmap updated to Team "Hanayamata" - Hana wa Odore ya Iroha ni Ho (TV Size) (Stixy) [Blossom]
```
[logs.zip](https://github.com/ppy/osu/files/8211668/logs.zip)

**Comment by Teages** [comment, marked as answer]
The problem founded: the discord overlay
